app/views/observations/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="col-md-12">
+  <h3>
+    Observations by Site
+    <%= link_to "Download CSV", observations_path(nil, format: :csv), class: "btn btn-success btn-xs" %>
+  </h3>
+  <% @by_site.each do |site, observations| %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4><%= site %></h4>
+      </div>
+        <div class="panel-body">
+          <% observations.each do |observation| %>
+          <h5>
+            <%= "Observation [Id: #{observation.id} Date: #{observation.date}]" %>
+            <%= link_to "Edit", edit_observation_path(observation), class: "btn btn-primary btn-xs" %>
+          </h5>
+          <table class="table table-striped table-bordered">
+            <thead>
+              <% observation.samples.first.response_data.each_with_index do |(k, v), i| %>
+                <% break if i > 6 %>
+                <th><%= k %></th>
```

Comment:
If you're just pulling the keys, it might be better to grab them from the protocol. That would also stop this page from crashing on edge case where the observation was started but nobody actually submitted any samples.